x11-wm/awesome/awesome-9999.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-LUA_COMPAT=( lua5-{1..3} luajit )
+LUA_COMPAT=( lua5-{1..4} luajit )
```

Comment:
But does ```lua5_3``` still work? Based on your description, it sounds, that it's not supported any more by current master?

Reply:
Yes as I tested, 5.1-5.3 doesn't work properly for the master and many dotfiles based on awesome-git.

Reply:
Well, in this case, you should remove lua5_{1..3} from LUA_COMPAT and only left lua5_4, if that's the only supported lua version.